package-lock.json

```diff
@@ -1,8 +1,33 @@
 {
```

Comment:
this file change seems like an accidental commit

Reply:
Hm, I can revert, but this is all autogenerated somehow.